config/crd/bases/confidentialcontainers.org_ccruntimes.yaml

```diff
@@ -14,6 +14,8 @@ spec:
     listKind: CcRuntimeList
     plural: ccruntimes
     singular: ccruntime
+    shotNames:
```

Comment:
typo ?

Reply:
Oops! I tested the branch "add_shortname_test" in my side but submitted the development branch "add_shortname" :(

Reply:
np :-)